docs/chart-and-series-types/sankey-diagram.md

```diff
@@ -1,49 +1,104 @@
 Sankey diagram
 ===
 
-A Sankey diagram is a type of flow diagram, in which the width of the link between two nodes is shown proportionally to the flow quantity.
+A Sankey diagram is a visualization that depicts a flow from one set of data to another. The sets of data are represented as nodes and the flow between them as links. The width of a connecting link between two nodes is proportional to the flow quantity.
 
-_For more detailed samples and documentation check the [API.](https://api.highcharts.com/highcharts/plotOptions.sankey)_
+_For more detailed samples and documentation, check the [API.](https://api.highcharts.com/highcharts/series.sankey)_
 
-<iframe style="width: 100%; height: 485px; border: none;" src=https://www.highcharts.com/samples/embed/highcharts/demo/sankey-diagram allow="fullscreen"></iframe>
+<iframe style="width: 100%; height: 485px; border: none;" src=https://www.highcharts.com/samples/highcharts/css/sankey allow="fullscreen"></iframe>
```

Comment:
If I understand the linked issue it is not about the style mode, but about the data structure in the article taken from the css demo. Maybe we should flip them so that `css/sankey` shows the countries in style mode and `demo/sankey-diagram` the ressources without style mode?